requirements.txt

```diff
@@ -103,36 +106,38 @@ django-parler==2.3
     #   django-parler-rest
 django-parler-rest==2.2
     # via django-munigeo
-django-reversion==5.0.1
+django-reversion==5.0.4
     # via -r requirements.in
-djangorestframework==3.13.1
+djangorestframework==3.14.0
     # via
     #   -r requirements.in
     #   django-orghierarchy
     #   django-parler-rest
     #   djangorestframework-bulk
     #   djangorestframework-gis
+    #   drf-oidc-auth
 djangorestframework-bulk==0.2.1
     # via -r requirements.in
 djangorestframework-gis==1.0
-    # via -r requirements.in
 djangorestframework-jwt==1.11.0
     # via -r requirements.in
-easy-thumbnails==2.8.3
+drf-oidc-auth==3.0.0
```

Comment:
Is this the version currently used in production? Just making sure, since I remember there at least used to be an issue when using newer than drf-oidc-auth==0.10.0 e.g. https://github.com/City-of-Helsinki/helerm/commit/ca652487af2753355851e00f9e8ad55a7b3a9fd9